src/user-guide/writing-workflows/runtime.rst

```diff
@@ -420,17 +420,59 @@ For this to work:
 Platforms, like other runtime settings, can be declared globally in the root
 family, or in other families, or for individual tasks.
 
+.. note::
 
-Dynamic Platform Selection
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+   The platform known as ``localhost`` is the platform where the scheduler
+   is running, in many cases a dedicated server and *not* your desktop.
+
+Internal Platform and Host Selection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :cylc:conf:`[runtime][<namespace>]platform` item points to either a
+:cylc:conf:`platform <global.cylc[platforms][<platform name>]>` or a
+:cylc:conf:`platform group <global.cylc[platform groups][<group>]>`.
+
+:term:`Platforms <platform>` are designed to allow sites to set sensible settings
+for compute resources with multiple login nodes.
+
+:term:`Platform groups <platform group>` represent groups of compute resources each
+of which is completely seperate, but where it does not matter which
+resource the scheduler runs a task-job on.
+
+:term:`Platforms <platform>` are selected from a :term:`platform group` once,
+when a job is submitted.
+
+Hosts within a :term:`platform` are re-selected each time the scheduler
+needs to communicate with a job.
+
+.. seealso::
+
+   :ref:`AdminGuide.PlatformConfigs`: For details of how Platforms and
+   Platform Groups are set up and in-depth examples.
+
+External Platform Selection Scripts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
```

Comment:
The functionality, or just say that it is here?

Reply:
There isn't a related configuration item so we don't have a good way to deprecate it in the code, lets cross that bridge when we have better platforms adoption.

We should say it here to avoid acquiring new users for this functionality. They should use platform groups instead (where appropriate, platform groups do not yet support load balancing which `rose host-select` does, however, we should add that in due course [easy])).